react/ContextHeader/styles.styl

```diff
@@ -14,10 +15,20 @@
   justify-content flex-start
   align-items center
 
+  +small-screen()
+      margin rem(8)
+
 .context-header-content
   flex-grow 1
+  flex-shrink 1
+  overflow hidden
+
+.context-header-text
```

Comment:
the `<Text/>` component already has a prop `ellipsis` to allow that, it shouldn't need extra styles for this.

Reply:
https://docs.cozy.io/cozy-ui/react/#text -> Props & Methods section

Reply:
Brillant, thanks! Missed that.